From 557b4261a3120b037e2c705ce94453bc63d08823 Mon Sep 17 00:00:00 2001
From: Otto Kekalainen <ottoke@amazon.com>
Date: Mon, 28 Nov 2022 21:03:50 -0800
Subject: [PATCH 3/4] WIP: Use `--replace_regex` in mysql-test-run to make SSL
 tests pass

Another potential area worthy of upstream contribution.

The new error messages _seem_ to follow uppercase and underscore format.

---
 mysql-test/main/ssl_crl.test | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mysql-test/main/ssl_crl.test b/mysql-test/main/ssl_crl.test
index 9b4758578a7..50f7d6981a0 100644
--- a/mysql-test/main/ssl_crl.test
+++ b/mysql-test/main/ssl_crl.test
@@ -9,5 +9,6 @@
 --echo # try logging in with a certificate in the server's --ssl-crl : should fail
 # OpenSSL 1.1.1a correctly rejects the certificate, but the error message is different
 --replace_regex /ERROR 2013 \(HY000\): Lost connection to server at '.*', system error: [0-9]+/ERROR 2026 (HY000): TLS\/SSL error: sslv3 alert certificate revoked/
+--replace_regex /SSLV3_ALERT_CERTIFICATE_REVOKED/sslv3 alert certificate revoked/
 --error 1
 --exec $MYSQL --ssl-ca=$MYSQL_TEST_DIR/std_data/cacert.pem --ssl-key=$MYSQL_TEST_DIR/std_data/client-key.pem --ssl-cert=$MYSQL_TEST_DIR/std_data/client-cert.pem test -e "SHOW STATUS LIKE 'Ssl_version'" 2>&1
-- 
2.39.2

